<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowWidget</class>
 <widget class="QMainWindow" name="MainWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISIS Reflectometry</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="mainTabs">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="helpButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBatch">
    <property name="title">
     <string>&amp;Batch</string>
    </property>
    <addaction name="newBatch"/>
    <addaction name="separator"/>
    <addaction name="loadBatch"/>
    <addaction name="saveBatch"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="showOptions"/>
    <addaction name="showSlitCalculator"/>
   </widget>
   <addaction name="menuBatch"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="newBatch">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="loadBatch">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="showOptions">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="showSlitCalculator">
   <property name="text">
    <string>&amp;Slit Calculator</string>
   </property>
  </action>
  <action name="saveBatch">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
